pages/activities.js

```diff
@@ -0,0 +1,85 @@
+import { Container, Navigation, Page } from "@/components/Layout/Layout";
+import ActivitiesList from "@/components/ActivitiesList/ActivitiesList";
+import styled from "styled-components";
+import { useState } from "react";
+import Guide from "@/components/Guide/Guide";
+import NavButton from "@/components/NavButton/NavButton";
+import PlusIcon from "@/assets/plus.png";
+import Image from "next/image";
+import ActivitiesForm from "@/components/ActivitiesForm/ActivitiesForm";
+import { motion } from "framer-motion";
+
+export default function Activities() {
+  const [showForm, setShowForm] = useState(false);
+  function handleShowForm() {
+    setShowForm(!showForm);
+  }
+
+  return (
+    <>
+      <Container>
+        <Page>
+          <Header>
+            <GradientText>
+              <Guide text={"activities"} />
+            </GradientText>
+            <NewActivity onClick={handleShowForm}>
+              <motion.div
+                animate={showForm ? { rotate: 45 } : {}}
+                style={{ height: "22px" }}
+              >
+                <Image
+                  width={22}
+                  height={22}
+                  src={PlusIcon}
+                  alt={"add new activity"}
+                />
+              </motion.div>
+            </NewActivity>
+          </Header>
+
+          {showForm ? (
+            <ActivitiesForm handleShowForm={handleShowForm} />
+          ) : (
+            <ActivitiesList showForm={showForm} />
+          )}
+        </Page>
+        {showForm ? null : (
+          <Navigation>
+            <NavButton linkToPage={"/"}>enter a mood</NavButton>
+          </Navigation>
+        )}
```

Comment:
why don't you integrate this ternary into the ternary above? in both cases you are checking if showForm is true, so you might as well combine these two :)

Reply:
es fiel ihm wie schuppen von den augen...